models/evaluator.py

```diff
@@ -0,0 +1,27 @@
+from nltk.translate.bleu_score import sentence_bleu
+
+
+def evaluation(translations, references):
+    """
+    evaluating a translation sentence against one or more reference sentences.
+    Args:
+        translations (string): translated sentence
+        references (list): list of reference sentences
+    Returns:
+        nothings for now, just print BLEU scores
+    """
+    print('translations=[%s], references=%s' % (translations, references))
```

Comment:
please put a `verbose` variable to whatever to do a print or not! printing all of those lists could be tricky
for example 
```python
def evaluation(...,..., verbose=False):
   if verbose:
         print("..............")
   ...
   ...
```